drafts/liberty-dev-mode-gradle.adoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Open Liberty development mode with Gradle"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/28316667
+author_github: https://github.com/rzgry
+seo-title: Open Liberty development mode with Gradle - OpenLiberty.io
+seo-description: Develop Java applications quicker using Open Liberty development mode with Gradle.
+blog_description: "Develop Java applications quicker using Open Liberty development mode with Gradle."
+---
+= Open Liberty development mode for Gradle
+Ryan Zegray <https://github.com/rzgry>
+
+Open Liberty development mode, or dev mode, now works with Gradle projects. Open Liberty dev mode allows you to develop applications with any text editor or IDE by providing hot reload and deployment, on demand testing, and debugger support. Your code is automatically compiled and deployed to your running server, making it easy to iterate on your changes. You can run tests on demand or even automatically so that you can get immediate feedback on your changes. You can also attach a debugger at any time to debug your running application.
+
+If you would like to try out dev mode with Gradle, you can follow the steps in the https://github.com/OpenLiberty/demo-devmode[demo-devmode] sample project.
```

Comment:
I'm not sure what the reader is expected to do here. If they follow the steps in the sample project, are they supposed to ignore the rest of this blog post? What does the blog post provide that the sample project doesn't, and vice versa?